include/mega65/fileio.h

```diff
@@ -82,7 +82,7 @@ chdir(char* filename);
 __attribute__((leaf))
 #endif
 uint8_t
-chdirroot(void);
+chdirroot(uint8_t partition);
```

Comment:
Update `tests/test-fileio.c` to reflect this breaking change. CI currently fails.